src/test/java/de/adorsys/keycloak/config/service/ImportAuthenticationFlowsIT.java

```diff
@@ -451,9 +452,14 @@ void shouldChangeBrowserFlow() throws IOException {
         AuthenticationExecutionExportRepresentation myForms2 = getExecutionFlowFromFlow(flow, "my forms 2");
         assertThat(myForms2, notNullValue());
         assertThat(myForms2.getRequirement(), is("ALTERNATIVE"));
-        assertThat(myForms2.getPriority(), is(4));
         assertThat(myForms2.isUserSetupAllowed(), is(false));
         assertThat(myForms2.isAutheticatorFlow(), is(true));
+
+        if (VersionUtil.ge(KEYCLOAK_VERSION, "25")) {
```

Comment:
Are there some changes regarding the order/priority in the flows in version 25?

Reply:
It seems like the flows get a priority behind the highest execution, which is 25. This seems more reasonable to me than just adding some kind of random numbers before Keycloak 25.